src/cf-typography/src/atoms/date.less

```diff
@@ -0,0 +1,6 @@
+.a-date {
```

Comment:
EDIT: In the event details as well => http://www.consumerfinance.gov/about-us/events/fall-2016-consumer-advisory-board-meeting-washington-dc/